pkg/bootstrapper/bootstrapper.go

```diff
@@ -400,7 +410,7 @@ func (wmcb *winNodeBootstrapper) createKubeletService() error {
 		"--cert-dir=" + certDirectory,
 		"--windows-service",
 		"--logtostderr=false",
-		"--log-file=" + filepath.Join(wmcb.installDir, "kubelet.log"),
+		"--log-file=" + filepath.Join(wmcb.logDir, "kubelet.log"),
```

Comment:
It seems like we should be using the log-dir option now
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

Reply:
I wanted to keep the change as less invasive as possible. I do not mind the log-dir option.

Reply:
Add a todo.

Reply:
How invasive is using `log-dir` option instead of `log-file`?

Reply:
`log-dir` creates different log files for info, error and warning, specific to each Process ID kubelet was associated with, along with some sym-links to the process. This will warranty the need for restructuring the tests to account for all the different files generated. This should be done as a part of [WINC-211](https://issues.redhat.com/browse/WINC-211)